tda/desktop_web/test_checkout.py

```diff
@@ -57,11 +61,14 @@ def test_checkout(desktop_web_driver, endpoints, batch_size, backend, random, sl
                 desktop_web_driver.find_element(By.CSS_SELECTOR, 'a[href="/checkout"]').click()
                 time.sleep(sleep_length())
 
-                desktop_web_driver.find_element(By.CSS_SELECTOR, '#email').send_keys("sampleEmail@email.com")
```

Comment:
I understand this is no longer necessary since you changed the app to pre-fill checkout address etc, right?
we never ended up changing that behavior depending on `se`?

Reply:
> I understand this is no longer necessary since you changed the app to pre-fill checkout address etc, right? 

Yep - when i was watching sauce video, i noticed it double filling email and the validation complaining. I'm not sure whether that caused any problems for sauce runs but made sense to remove it

> we never ended up changing that behavior depending on `se`?

Could you clarify what you mean here? I don't understand

Reply:
I don't remember the details (probably happened on slack) but vaguely remember that initial change (https://github.com/sentry-demos/empower/pull/253) broke TDA and was later fixed with https://github.com/sentry-demos/empower/commit/3493cfead5b0803d5614b08f712779c69044377f